index.html

```diff
@@ -129,46 +129,57 @@ <h2>
                     <tr>
                         <th>Time</th>
                         <th>Schedule</th>
+                        <th>Location</th>
                     </tr>
                     <tr>
                         <td>9:15 AM</td>
                         <td>Registration and Morning Coffee</td>
+                        <td>Activities & Recreation Center (ARC)</td>
```

Comment:
```suggestion
                        <td>North Creek Events Center</td>
```
this will be in the NCEC